automatic/ssis-vs2019/tools/ChocolateyInstall.ps1

```diff
@@ -0,0 +1,14 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageArgs = @{
+    packageName   = $env:ChocolateyPackageName
+    softwareName  = 'Microsoft.DataTools.IntegrationServices*'
+    fileType      = 'exe'
+    silentArgs    = "/quiet /log `"$env:TEMP\$env:ChocolateyPackageName.$env:ChocolateyPackageVersion.log`""
+    validExitCodes= @(0)
+    url           = "update"
+    checksum      = "update"
+    checksumType  = "sha256"
+  }
+  
+Install-ChocolateyPackage @packageArgs
```

Comment:
We might want to check if `devenv.exe` is running before trying to install to give a better error message. I created #56 for tracking this.